public/app/plugins/panel/bargauge/BarGaugePanel.tsx

```diff
@@ -100,8 +100,10 @@ export class BarGaugePanel extends PureComponent<PanelProps<BarGaugeOptions>> {
         renderCounter={renderCounter}
         width={width}
         height={height}
+        minVizHeight={10}
         itemSpacing={this.getItemSpacing()}
         orientation={options.orientation}
+        repeaterStyle={{ overflow: 'hidden scroll' }}
```

Comment:
Uhm, maybe this style could be set internally in the VizRepeater when `minVizHeight` is set?

Reply:
Oh good idea, will do